source/css/scss/vendor_override/foundation/_panels.scss

```diff
@@ -0,0 +1,5 @@
+.panel {
+	&.callout {
+		border-color: scale-color($callout-panel-bg, $lightness: -11%);
```

Comment:
While I have no objection to using scale-color as it's a perfectly legit tool, there is no active use of it that I see in our styleguide scss thus far. (All the uses that came in from default Foundation scss are commented out.) And there is an argument to be made that all our UI colors should come from a defined palette represented in http://styleguide.kiva.org/?p=atoms-colors rather than ad-hoc values set in a particular element's style. Doing so is similar to setting an element-local type size rather than extending our predefined typography set if a use case requires something not already provided for. If this is too much of a "fuss-budget" comment, feel free to ignore it.